### PR TITLE
add some comments to provide more information regarding the http 401 error

### PR DESCRIPTION
**Pull request by colinwerner**
I added some comments to the code regarding my change.

Please ignore all the whitespace changes.

### CODE REVIEW

File: app/misc/authenticate.ts [hunk 1]
```diff
@@ -138,13 +138,15 @@ function getUserInfo(callback) {
           $("#otpModal").modal('show');
         });
       }
+      // HttpError has the error code in statusCode
```
Comment (colinwerner) [inline app/misc/authenticate.ts]: change 1

File: app/misc/authenticate.ts [hunk 2]
```diff
@@ -138,13 +138,15 @@ function getUserInfo(callback) {
...
       else if (401 == err.statusCode) {
+        // 401 ==> Unauthorized (hence invalid username and password)
```
Comment (colinwerner) [inline app/misc/authenticate.ts]: change 2

**Comment by colinwerner** [comment]
This is part of the fix for issue #14.

File: app/misc/authenticate.ts [hunk 3]
```diff
@@ -107,8 +107,8 @@ function searchRepoName() {
 
 function getUserInfo(callback) {
 
-  
-  if (signedAfter === true){  // if the trys to login after clicking "continues without sign in" 
+
+  if (signedAfter === true){  // if the trys to login after clicking "continues without sign in"
```
Comment (mehrler12) [inline app/misc/authenticate.ts]: I know you didn't write this one but since we're working on comments this one doesn't make any sense.

Comment (colinwerner) [reply app/misc/authenticate.ts]: I fixed that as well, will merge now.

**Pull request merged**